screenfetch.ps1

```diff
@@ -49,86 +53,88 @@ $CPU = (((gwmi Win32_Processor).Name) -replace '\s+', ' ');
 $GPU = (gwmi Win32_DisplayConfiguration).DeviceName;
 $FreeRam = ([math]::Truncate((gwmi Win32_OperatingSystem).FreePhysicalMemory / 1KB)); 
 $TotalRam = ([math]::Truncate((gwmi Win32_ComputerSystem).TotalPhysicalMemory / 1MB));
-$RamPercent = ($FreeRam / $TotalRam) * 100;
-$RamPercent = "{0:N0}" -f $RamPercent;
-
+$UsedRam = $TotalRam - $FreeRam;
+$FreeRamPercent = ($FreeRam / $TotalRam) * 100;
+$FreeRamPercent = "{0:N0}" -f $FreeRamPercent;
+$UsedRamPercent = ($UsedRam / $TotalRam) * 100;
+$UsedRamPercent = "{0:N0}" -f $UsedRamPercent;
 
 ####### Printing Output #########
 
 # Line 1 - HostName
-Write-Host "                         ....::::       " -f Cyan -NoNewline;
+Write-Host "                          ,,,;;;1       " -f Cyan -NoNewline;
 Write-Host $Username -f red -nonewline; 
 Write-Host "@" -f gray -nonewline; 
 Write-Host $Machine -f red;
 
 # Line 2 - OS
-Write-Host "                 ....::::::::::::       " -f Cyan -NoNewline;
+Write-Host "                ,,;;;101000100001       " -f Cyan -NoNewline;
 Write-Host "OS: " -f Red -NoNewline;
 Write-Host $OS $BitVer;
 
 # Line 3 - Kernel
-Write-Host "        ....:::: ::::::::::::::::       " -f Cyan -NoNewline;
+Write-Host "     ,,,;;;11   10101001001010010       " -f Cyan -NoNewline;
 Write-Host "Kernel: " -f Red -nonewline;
 Write-Host $Kernel;
 
 # Line 4 - Uptime
-Write-Host "....:::::::::::: ::::::::::::::::       " -f Cyan -NoNewline;
+Write-Host "0111101000111   10110111101101110       " -f Cyan -NoNewline;
 Write-Host "Uptime: " -f Red  -nonewline;
 Write-Host $uptime.Days"d " $uptime.Hours"h " $uptime.Minutes"m " $uptime.Seconds"s " -separator "";
 
 # Line 5 - Motherboard
-Write-Host ":::::::::::::::: ::::::::::::::::       " -f Cyan -NoNewline;
+Write-Host "1111010110101   10010100101011010       " -f Cyan -NoNewline;
 Write-Host "Motherboard: " -f Red -nonewline; 
 Write-Host $Motherboard.Manufacturer $Motherboard.Product;
 
 # Line 6 - Shell (Hardcoded since it is unlikely anybody can run this without powershell)
-Write-Host ":::::::::::::::: ::::::::::::::::       " -f Cyan -NoNewline;
+Write-Host "0101010001010   01010101000101010       " -f Cyan -NoNewline;
 Write-Host "Shell: " -f Red -nonewline; 
-Write-Host "Powershell 3.0"
+Write-Host "PowerShell $($PSVersionTable.PSVersion.ToString())"
 
 # Line 7 - Resolution (for primary monitor only)
-Write-Host ":::::::::::::::: ::::::::::::::::       " -f Cyan -NoNewline;
+Write-Host "1010101010110   01100001011110100       " -f Cyan -NoNewline;
```

Comment:
For all cases, mind leaving out these changes in this PR? I want the art to eventually go through a generator where the user can choose the ascii art with a simple param flag